src/Microsoft.Health.Fhir.Shared.Web/roles.json

```diff
@@ -40,7 +40,8 @@
                 "*"
             ],
             "notDataActions": [
-                "hardDelete"
+                "hardDelete",
+                "profileAdmin"
```

Comment:
Would it be better to add a separate role for ProfileAdmin so that a user can be given profile editing permissions without having to be given global admin role?

Reply:
Maybe... I actually not sure, is this roles.json set in stone for PaaS, or they can be modified, and this is just sample of possible assignment?

Reply:
Not sure. I think @johnstairs would know the answer to that.